infra/prod/index.ts

```diff
@@ -80,7 +80,7 @@ const postgresql = new aws.rds.Instance(
     allocatedStorage: 10,
```

Comment:
```suggestion
    allocatedStorage: 20,
    maxAllocatedStorage: 100,
```

- 20 GB is the storage the is currently allocated for this DB (the change was made manually)
- autoscaling is also enabled, having `maxAllocatedStorage` set in pulumi will enable autoscaling in the RDS if I understand correctly